includes/template-library/data/endpoints/templates.php

```diff
@@ -0,0 +1,20 @@
+<?php
+namespace Elementor\Includes\TemplateLibrary\Data\Endpoints;
+
+use Elementor\Data\Base\Endpoint;
+use Elementor\Plugin;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+class Templates extends Endpoint {
+	public function get_name() {
+		return 'templates';
+	}
+
+	public function get_items( $request ) {
+		return Plugin::$instance->templates_manager->get_library_data( [ 'only_sources', $request->get_param( 'source' ) ] );
```

Comment:
What does this string 'only sources' mean? should it be only `[$request->get_param('source')]`? or `[ 'only_sources' => $request->get_param('source') ]`?